benches/coprocessor_executors/mod.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 PingCAP, Inc.
+// Copyright 2018 The TiKV Authors.
```

Comment:
Is `the` necessary?

Reply:
/cc @queenypingcap

Reply:
We discussed on Slack and the conclusion was that we should remove the copyright line entirely, but leave the Apache license header (it seems that CNCF has no policy or precedent and Apache recommend (but don't require) the license header and no copyright.

Reply:
We decided on having:

```
// Copyright xxxx TiKV Project Authors
```

(https://pingcap.slack.com/archives/C2Q2F63SM/p1553818611014100)

Reply:
Better be:
```
// Copyright xxxx TiKV Project Authors. Licensed under Apache-2.0.
```
, so if someone can only see one file, they can still reuse it without worrying about the license.

Reply:
Yes a great idea.